command/git.go

```diff
@@ -99,6 +150,7 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 	errChan := make(chan error)
 	go func() {
 		logger.Execution().Info("Fetching source from git...")
+		logger.Execution().Debug(fmt.Sprintf("Commands are: %s", cmdsJoined))
```

Comment:
do we risk leaking tokens this way?

Reply:
Yes. Should i just remove this completely?